IHP/ValidationSupport/ValidateField.hs

```diff
@@ -128,7 +166,27 @@ nonEmpty value | null value = Failure "This field cannot be empty"
 nonEmpty _ = Success
 {-# INLINE nonEmpty #-}
 
--- | Validates value looks like a phone number
+
+-- | Validates that value is empty
+--
+-- >>> empty "hello world"
+-- Failure "This field must be empty"
+--
+-- >>> empty ""
+-- Success
+--
+-- >>> empty (Just "hello")
+-- Failure "This field must be empty"
+--
+-- >>> empty Nothing
+-- Success
+empty :: MonoFoldable value => value -> ValidatorResult
+empty value | null value = Success
+empty _ = Failure "This field must be empty"
```

Comment:
I would suggest naming this `isEmpty` to avoid name conflicts inside the controller actions (e.g. when you have a local variable `empty` in scope. It's also usually used for data structures, [e.g. Data.Map exports a `empty` function](https://hackage.haskell.org/package/containers-0.4.0.0/docs/Data-Map.html)).

Reply:
I agree, and I'd prefer `isNonEmpty` as well. I had that change originally but ran into the issue that `IHP.Prelude` exports `isEmpty` already. Any suggestions for how to handle this?

Reply:
Maybe we can call it `isEmptyValue`? :)

Regarding `nonEmpty`: After think about it, I would keep the `nonEmpty` because `isNonEmptyValue` would be very long, and the `nonEmpty` validator is used really often.

Reply:
:+1: for pragmatism! I've changed the code accordingly.